cmake/GrpcTargets.cmake

```diff
@@ -147,6 +147,9 @@ function(generate_grpc_files)
 
     set(did_generate_proto_sources FALSE)
     if("${newest_proto_dependency}" IS_NEWER_THAN "${GENERATED_PROTO_DIR}/${path_base}.pb.cc")
+      # resolve root_path, proto_file to real path's - protoc check that root_path is prefix of proto_file (this can be non true if project inside folder sym linked to other dir)
+      get_filename_component(real_root_path ${root_path} REALPATH)
```

Comment:
A few lines above we already do this, so the root_path should be already a real path. Are we missing something?